h/redirects

```diff
@@ -0,0 +1,7 @@
+# Legacy application URLs
```

Comment:
Since the application will fail to boot when this file is missing, maybe we should add a comment at the top saying `# For the application to boot, this file needs to exist, even if it's just empty`. Just so that when we (for some reason) in the future remove all redirects that we remember to keep the empty file around instead of just deleting it.